building/devices/stm32mp1.rst

```diff
@@ -45,14 +45,14 @@ as listed in table below:
 +========================+======================================+
 | `STM32MP135F-DK`_      | ``PLATFORM=stm32mp1-135F_DK``        |
 +------------------------+--------------------------------------+
-| `STM32MP157A-DK1`      | ``PLATFORM=stm32mp1-157A_DK1``       |
-| `STM32MP157D-DK1`      |                                      |
+| `STM32MP157A-DK1`_     | ``PLATFORM=stm32mp1-157A_DK1_SCMI``  |
+| `STM32MP157D-DK1`_     |                                      |
 +------------------------+--------------------------------------+
-| `STM32MP157C-DK2`      | ``PLATFORM=stm32mp1-157C_DK2``       |
-| `STM32MP157F-DK2`      |                                      |
+| `STM32MP157C-DK2`_     | ``PLATFORM=stm32mp1-157C_DK2_SCMI``  |
+| `STM32MP157F-DK2`_     |                                      |
 +------------------------+--------------------------------------+
-| `STM32MP157C-EV1`      | ``PLATFORM=stm32mp1-157C_EV1``       |
-| `STM32MP157F-EV1`      |                                      |
+| `STM32MP157C-EV1`_     | ``PLATFORM=stm32mp1-157C_EV1_SCMI``  |
+| `STM32MP157F-EV1`_     |                                      |
```

Comment:
I think we should let the non-SCMI variant appear as it is still supported

Reply:
Ok, i'll add them back...